ivy/functional/frontends/torch/tensor.py

```diff
@@ -1693,6 +1693,23 @@ def gcd(self, other, *, out=None):
     def isnan(self):
         return torch_frontend.isnan(self)
 
+    @with_unsupported_dtypes(
+        {
+            "2.0.1 and below": (
+                "bfloat16",
+                "int8",
+                "uint8",
+                "int16",
+                "complex128",
+                "complex64",
+            )
+        },
+        "torch",
+    )
+    def triu_(self, diagonal=0):
+        self.ivy_array = self.triu(diagonal).ivy_array
```

Comment:
There is no method named `triu` implemented in this class which is causing all the tests to fail. Instead use `torch_frontend.triu` here

Reply:
@NiteshK84 Did you forget to make this change? There is no method named `triu` in this class and so rather than adding it yourself, just use the `torch_frontend.triu` here as mentioned in my comment above.